src/client.ts

```diff
@@ -0,0 +1,32 @@
+
+import axios from "axios"
+import datos from "./datos.json"
+
+
+async function postData() {
+  axios.post('http://95.217.235.69/',datos)
+  .then(function(response){
+    console.log(response);
+  }).catch(function(error){
+    console.log(error);
+  })
+}
+
+
+function getData() {
+  axios.get(`http://95.217.235.69/${process.env.EMAIL}`,{
+    
+    
+    headers:{
+      "Content-Type": "application/json",
+      "x-password": process.env.PASSWORD,
```

Comment:
```suggestion
  axios.get("http://95.217.235.69/"+data.contactInfo.emailAddress,{
    
    headers:{
      "Content-Type": "application/json",
      "x-password": data.credentials.password,
```
Para mantenerse consistente, y no causar mal entendidos; como que `process.env.EMAIL` en `GET` no es igual a  `data.contactInfo.emailAddress` en `POST`, y que no estamos buscando dos cosas totalmente diferente.

**Sugerencia:** podemos utilizar la misma variable importada de `datos` para llenar los parámetros tanto del `POST `como el `GET`.